.github/release-drafter.yml

```diff
@@ -31,8 +31,4 @@ version-resolver:
   patch:
     labels:
       - 'version:patch'
-  default: patch
-template: |
```

Comment:
I think this `template` is mandatory. The action will use `Changes` as an h1 header in the notes. This string can be anything you'd like.  And you can also include any other strings after that.

`$CHANGES` is a replacement anchor that tells the action where to place the release notes.